app/src/modules/content/routes/collection.vue

```diff
@@ -453,6 +454,11 @@ export default defineComponent({
 			await layoutRef.value?.state?.refresh?.();
 		}
 
+		async function saveAndRefresh() {
```

Comment:
This function doesn't save anything. Lets remove "save" from the name here 👍🏻

Reply:
The naming stemmed from the refresh only happening in the save function of the batch drawer, but on hindsight it does make sense to not add it in the name.

Changed to `drawerBatchRefresh` now since it's specific to the drawer-batch here.

Reply:
When can this be released?

Reply:
There is no specific timeline set in stone for this specific PR as of now, as the focus was diverted to Directus cloud, hence the delay of PR reviews/merges as of late. With that said, PR reviews should pick up more traction going forward, so hopefully that's understandable 👍

Reply:
No pressure, just waiting for this fix.